soc/xtensa/intel_adsp/Kconfig.defconfig

```diff
@@ -13,7 +13,7 @@ config XTENSA_CACHED_REGION
 config XTENSA_UNCACHED_REGION
 	default 4
 
-if SOC_FAMILY_INTEL_ADSP
+if CONFIG_INTEL_ADSP_CAVS
```

Comment:
the CONFIG_ prefix isn't needed here, the endif comment should match the INTEL_ADSP_CAVS label as well